app/representers/api/msg/series_representer.rb

```diff
@@ -0,0 +1,11 @@
+# encoding: utf-8
+
+class Api::Msg::SeriesRepresenter < Api::SeriesRepresenter
+  # don't embed the stories, makes this a much smaller message
+  embed :public_stories,
+        as: :stories,
+        paged: true,
+        item_class: Story,
+        item_decorator: Api::Min::StoryRepresenter,
+        zoom: false
```

Comment:
that's the idea, but since this is used in the announcement, there is no param passed in, so effectively this will mean it won't ever be embedded.